commercial/app/views/hosted/guardianHostedArticle.scala.html

```diff
@@ -43,7 +43,7 @@
 
     </style>
     <!--<![endif]-->
-    @guardianHostedHeader(if(page.campaign.fontColour.shouldHaveBrightFont) "hosted-article-page hosted-page--bright" else "hosted-article-page", page.campaign.logo.url, page.campaign.owner, page.campaign.logoLink.getOrElse(page.cta.url))
```

Comment:
I think this `page.campaign.logoLink.getOrElse(page.cta.url)` is here so that you can choose different links for the logo and the CTA, not sure any of the campaigns actually has done that though?